### PR TITLE
refactor : make cart page responsive

### PR DESCRIPTION
**Pull request by viru2001**
## make cart page responsive

responsive-cart page

[Deploy Preview](https://deploy-preview-12--gamers-zone.netlify.app/pages/cart.html)

### CODE REVIEW

File: styles/cart.css [hunk 1]
```diff
@@ -54,3 +58,29 @@ footer {
...
+      "details"
+      "footer";
+  }
+  #order-details{
+    display: block;
+    margin: 0 auto;
+  }
+}
+
+@media (max-width:624px) {
```
Comment (anupkrbid) [inline styles/cart.css:78]: prefer using ems over px

File: styles/cart.css [hunk 2]
```diff
@@ -54,3 +58,29 @@ footer {
   top: 20rem;
   height: fit-content;
 }
+
+@media (max-width: 1038px) {
```
Comment (anupkrbid) [inline styles/cart.css:62]: prefer using relative units in media query

**Review by anupkrbid — APPROVED** [review]
Looks good. 
Just one suggestion

File: styles/cart.css (hunk 2 above)
Comment (viru2001) [reply styles/cart.css:62]: ok will update all media queries at the end

**Review by SHIKHASINGH1506 — APPROVED** [review]
If possible make media query units relative. That's all. Good to merge for now

**Pull request merged**